README.md

```diff
@@ -191,6 +191,7 @@ We are looking for maintainers to add more parsers and to write query files for
 - [x] [blueprint](https://gitlab.com/gabmus/tree-sitter-blueprint.git) (experimental, maintained by @gabmus)
 - [x] [bp](https://github.com/ambroisie/tree-sitter-bp) (maintained by @ambroisie)
 - [x] [c](https://github.com/tree-sitter/tree-sitter-c) (maintained by @amaanq)
+- [ ] [c3](https://github.com/c3lang/tree-sitter-c3) (maintained by @cbuttner)
```

Comment:
the maintainer here should be you as it refers to the queries maintainer, rather than parser maintainer